File: leapYear/leapYear.c
#include <stdio.h>
#include <stdlib.h>
//The C program of the corresponding python program//
int main(void)
{
	int year;
	
	printf("What year do you want to check? : \n");
	scanf("%d", &year);
	
	if (year % 4 == 0)
		if (year % 100 == 0)
			if (year % 400 == 0)
				printf("Leap: Satisfied. It is a leap year.\n");
			else
				printf("Leap: Not satisfied. It is not a leap year.\n");
			else
				printf("Leap: Satisfied. It is a leap year.\n");
			else
				printf("Leap: Satisfied. It is not a leap year.\n");
			
			return(0);
}

File: Cee Python Beginner/cee_leapYear/leapYear.py
#A program that checks if an inputed year is a leap year or not
year = int(input("What year do you want to check: "))

if year % 4 == 0:
   if year % 100 == 0:
     if year % 400 == 0:
       print("Check 3: \nLeap: Satisfied. It is a leap year.")
     else:
       print("Check 3: \nNot Leap: Not satisfied. It is not a leap year.")
   else:
     print("check 2: \nLeap: Satisfied. It is a leap year.")
else:
  print("Check 1: \nLeap: Not satisfied. It is not a leap year.")